operate/schema/src/main/java/io/camunda/operate/store/opensearch/client/sync/OpenSearchDocumentOperations.java

```diff
@@ -111,10 +111,23 @@ private <R> Map<String, Aggregate> scrollWith(
       final Consumer<HitsMetadata<R>> hitsMetadataConsumer,
       final Class<R> clazz)
       throws IOException {
+    final SearchResponse<R> response = openSearchClient.search(request, clazz);
```

Comment:
I only added final, I can revert if we want have more clear PR. (But adding by our style approach =))

Reply:
ah so it did not come from Optimize presumably? I was worried that we had resolved a conflict in the wrong place, for example. I would say we should not touch Operate files in this PR. I think it will even trigger the Operate CI on each push, and we don't really want that. If the change is only small, let's exclude it here 👍

Reply:
fixed